cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/BatchPartitionConsumer.java

```diff
@@ -55,7 +58,8 @@
    * will return the paths of the Partitions that have yet to be consumed, according to the state deserialized from
    * the implementing readBytes(DatasetContext) method.
    */
-  public PartitionedFileSet getPartitionedFileSet(DatasetContext datasetContext, String partitionedFileSetName) {
+  public PartitionedFileSet getPartitionedFileSetForConsuming(DatasetContext datasetContext,
```

Comment:
I am still not a fan of this method name... let's brainstorm....
perhaps getConfiguredDataset()? Or configureAndGetDataset()? or just configureDataset()?